src/classes/CON_PrimaryAffls_TEST.cls

```diff
@@ -952,12 +1389,348 @@ private with sharing class CON_PrimaryAffls_TEST {
+
+        Contact reportToContact = UTIL_UnitTestData_TEST.getUniqueContact(contactCount+5);
+        insert reportToContact;
+
+        List<Contact> contactsToBeInserted = new List<Contact>();
+        for (integer i = 0; i < contactCount; i++) {
+            Contact con = UTIL_UnitTestData_TEST.getUniqueContact(i);
+            con.ReportsToId = reportToContact.Id;
+            contactsToBeInserted.add(con);
+        }
+
+        String errorMessageText = CON_PrimaryAffls_TEST.getInvalidRecordTypeInAffiliationMappingMessage('Business_Organization_Invalid');
+
+        Test.startTest();
+        try{
+            insert contactsToBeInserted;
+        } catch (System.DmlException ex){
+            System.assertEquals(true, ex.getMessage().contains(errorMessageText), 'An invalidRecordTypeInAffiliationMapping error should be thrown.');
+        }        
+        Test.stopTest();
+    }
+
+    /****************************************************************************************************************
+    * @description Tests to ensure that when a Contact is inserted with a primary affiliation field populated,  the
+    * mapping is to a field that is not of type lookup to Account, Record Type mapping is invalid and validation is 
+    * disabled, an invalidContactFieldInAffiliationMapping error is thrown.
+    **************************************************************************************************************/
+    @isTest
+    private static void insertContactWithInvalidRTMappingAndInvalidFieldMappingToAccountLookupValidationOff() {
+        UTIL_CustomSettings_API.getSettingsForTests(new Hierarchy_Settings__c(Affiliation_Record_Type_Enforced__c = false));       
+        
+        // insert mappings
+        String namespace = UTIL_Namespace.getNamespace();
+        String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
+
+        List<Affl_Mappings__c> mappings = new List<Affl_Mappings__c>();
+        mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Business_Organization_Invalid', Primary_Affl_Field__c = prefix + 'ReportsToId'));
+        mappings.add(new Affl_Mappings__c(Name = 'Household Account', Account_Record_Type__c = 'HH_Account', Primary_Affl_Field__c =  prefix + 'Primary_Household__c'));
+        insert mappings;
+
+        Contact reportToContact = UTIL_UnitTestData_TEST.getUniqueContact(contactCount+5);
+        insert reportToContact;
+
+        List<Contact> contactsToBeInserted = new List<Contact>();
+        for (integer i = 0; i < contactCount; i++) {
+            Contact con = UTIL_UnitTestData_TEST.getUniqueContact(i);
+            con.ReportsToId = reportToContact.Id;
+            contactsToBeInserted.add(con);
+        }
+
+        String errorMessageText = CON_PrimaryAffls_TEST.getinvalidContactFieldInAffiliationMappingMessage('ReportsToId');
+
+        Test.startTest();
+        try{
+            insert contactsToBeInserted;
```

Comment:
Another consideration of the single-line test block and the `System.assert(false,String)`